### PR TITLE
Guidance for 121s

### PR DESCRIPTION
**Pull request by yaseminercan**
A first version of basic 121 guidance. This speaks to Line Managers and the people who they line manage. Here called 'Direct reports' for lack of a better word.

The aim is to keep this guidance as brief as possible. 
There are of course more things we use 121s for, such as building a good relationship and get to know each other and check in on peoples well-being. 
I think this page could quickly become a long read so wanted to just get the basics written up here. Thoughts?

### CODE REVIEW

**Review by kjdchapman — COMMENTED** [review]
I have ideas! This looks good though - excited to get more of this in the handbook!

File: guides/line-management/one-to-ones.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# One-to-one Meetings
+
+Our Line Managers meet frequently with their 'direct reports' for a minimum of 1 hour per month. 
+
+During probation we recommend a 30 minute meeting every week or fortnightly to help a new person on-board successfully into Made Tech.
```
[the remark this answers is not in the document]
Comment (yaseminercan) [reply guides/line-management/one-to-ones.md:5]: That sounds reasonable. I wonder how often line managers go to read about probation so want to make its captured in the right place. happy to replace with a link to probation. Does that work? (once it exists :D )

File: guides/line-management/one-to-ones.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# One-to-one Meetings
+
+Our Line Managers meet frequently with their 'direct reports' for a minimum of 1 hour per month. 
+
+During probation we recommend a 30 minute meeting every week or fortnightly to help a new person on-board successfully into Made Tech. 
+
+
+## What are 121's used for?
```
[the remark this answers is not in the document]
Comment (yaseminercan) [reply guides/line-management/one-to-ones.md:8]: We refer to them internally as 121s. Think its best to reflect reality in my opinion. What do others think?

File: guides/line-management/one-to-ones.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+# One-to-one Meetings
+
+Our Line Managers meet frequently with their 'direct reports' for a minimum of 1 hour per month.
```
[the remark this answers is not in the document]
Comment (yaseminercan) [reply guides/line-management/one-to-ones.md:3]: Happy to remove the quotes :) still don't like the wording direct report

[the remark this answers is not in the document]
Comment (yaseminercan) [reply guides/line-management/one-to-ones.md:3]: Agreed!

File: guides/line-management/one-to-ones.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
+# One-to-one Meetings
+
+Our Line Managers meet frequently with their 'direct reports' for a minimum of 1 hour per month. 
+
+During probation we recommend a 30 minute meeting every week or fortnightly to help a new person on-board successfully into Made Tech. 
+
+
+## What are 121's used for?
+121's are dedicated time for the Line Manager and their 'direct reports' to discuss and cover:
+- clarity of their role
```
[the remark this answers is not in the document]
Comment (yaseminercan) [reply guides/line-management/one-to-ones.md:10]: I like it!

File: guides/line-management/one-to-ones.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+- career progression 
+- feedback
+- mentoring/coaching
+
+
+## Our 121 tool
+We use a tool called Small Improvements to schedule and take notes of all our 121's. These notes become data points that 'direct reports' can use to prepare for annual reviews and promotions.
+A scheduled 121 will have a list of 'Talking points' (agenda) that you can setup in advance.
+It's also an easy way for 'direct reports' to ask peers for feedback and set objectives for career progression.
```
[the remark this answers is not in the document]
Comment (yaseminercan) [reply guides/line-management/one-to-ones.md:19]: You get access to Small Improvements during on-boarding yes.
I thought that referencing this here is important as things like 'Talking points' is specific to that platform. If we did change platforms then we may also have to change naming conventions at the same time

File: guides/line-management/one-to-ones.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+- record 121s and notes on Small Improvements
+- provide clarity on role expectations and performance
+- help prepare 'direct reports' work torwards career progression (or passing probation)
+- share and encourage feedback
+
+## 'Direct report' 121 expectations
+- take ownership of setting and reviewing goals frequently
+- set 'Talking points' ahead of 121's
+- prepare examples of work or feedback you want to talk about
+- take notes in Small Improvements
```
[the remark this answers is not in the document]
Comment (yaseminercan) [reply guides/line-management/one-to-ones.md:33]: Its an expectation that direct reports should aim to get to.
We should coach direct reports to, over time, take ownership of their career and goal setting. so we can step back into a coaching role and empower them to take it further, with guidance from us. 
This becomes more and more important throughout their career and as they rise into senior roles so starting this early will help them a lot over time.

File: guides/line-management/one-to-ones.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (StevenLeighton21) [reply guides/line-management/one-to-ones.md:5]: Suggest rewording to 'There may be periods where you meed your direct reports more frequently or for more time per month, such as during their probation period.'

**Review by StevenLeighton21 — APPROVED** [review]
Looks good to me - I'd vote for merging and iterating as this has been open for a month

File: guides/line-management/one-to-ones.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (yaseminercan) [reply guides/line-management/one-to-ones.md:5]: Thanks Steve, sounds good. Do you mean 'for longer time per month' as in more than 1 hour?

[the remark this answers is not in the document]
Comment (StevenLeighton21) [reply guides/line-management/one-to-ones.md:5]: yeah that's what I meant :)
